pkg/kubelet/preemption/preemption_test.go

```diff
@@ -85,6 +86,9 @@ func getTestCriticalPodAdmissionHandler(podProvider *fakePodProvider, podKiller
 }
 
 func TestEvictPodsToFreeRequests(t *testing.T) {
+	if err := utilfeature.DefaultFeatureGate.Set("ExperimentalCriticalPodAnnotation=true"); err != nil {
+		t.Errorf("failed to set ExperimentalCriticalPodAnnotation to true: %v", err)
+	}
```

Comment:
curious how was it working before?

Reply:
took me a second to figure it out.  the `IsCriticalPod()` in `sortPodsByQOS()` was unprotected by the feature gate.